include/boost/url/params.hpp

```diff
@@ -164,13 +164,26 @@ class params
         <!-- @{ -->
      */
 
+    /** Copy Assignment
+
+        This instance of @ref params will point to the other url.
+
+        @return Reference to this instance of @ref params
+
+        @param other Other params to get the url from
+
+     */
+    inline
```

Comment:
I thought we were putting `inline` on the definitions and not the declarations?

Reply:
There's no definition because the declaration is just `= default`.

Reply:
Oh, yeah, didnt' see that. Then we don't need `inline` at all, because definitions which appear in the declaration are inline by default, and a defaulted member function counts as a definition.